dali/operators/reader/webdataset_reader_op.cc

```diff
@@ -94,6 +94,9 @@ The index file can be generated using a dedicated script::
 
     ``<path_to_dali>/tools/wds2idx.py <path_to_archive> <path_to_index_file>``
 
+If the index file is not provided, it will be automatically inferred from the tar file.
```

Comment:
```suggestion
If the index file is not provided, it will be automatically generated from the tar file.
```

Reply:
I'd rather use inferred because the index is not saved anywhere afterwards